types/ember-data/test/belongs-to.ts

```diff
@@ -27,3 +27,5 @@ folder.get('parent').then(parent => {
 folder.set('parent', folder);
 folder.set('parent', folder.get('parent'));
 folder.set('parent', store.findRecord('folder', 3));
+
+assertType<DS.Model | null>(folder.belongsTo('parent').value());
```

Comment:
This test would pass, even if `value()` had no possibility of returning null

Reply:
I've changed it to use `// $ExpectType`. I'm unsure if that's the best approach though.
At least it'll now error when removing the null return `value()`.

Can you maybe point me to a better solution of testing this, if there is one?

Reply:
`$ExpectType` is the thing to use here -- you're asserting that some type is exactly what you expect, rather than whether values of that type are assignable to some other type (i.e., an `any` could very easily slip through)